output/components.eslint.txt

```diff
@@ -19,10 +19,6 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   33:11  error  Caution: `i18next` also has a named export `t`. Check if you meant to write `import {t} from 'i18next'` instead  import/no-named-as-default-member
   35:11  error  Caution: `i18next` also has a named export `t`. Check if you meant to write `import {t} from 'i18next'` instead  import/no-named-as-default-member
 
-/home/travis/build/Talend/ui/packages/components/src/DateTimePickers/generator.js
-  3:8  error  'differenceInCalendarWeeks' is defined but never used  no-unused-vars
-  5:8  error  'endOfMonth' is defined but never used                 no-unused-vars
-
```

Comment:
these 2 methods have been deleted already, but this components.eslint.txt is not updated.